### PR TITLE
Document using generators in Deployments

### PR DESCRIPTION
**Pull request by xordspar0**
This adds documentation for how to use resources made by configMapGenerators and secretGenerators after running kustomize. Existing documentation explained how to create these resources, but not how to use them (unless I missed it).

### CODE REVIEW

**Comment by xordspar0** [comment]
I've signed the CLA.

File: content/en/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 1]
```diff
@@ -170,6 +257,53 @@ metadata:
 type: Opaque
 ```
 
+Generated Secrets can be used in Deployments by refering to the name of the secretGenerator:
+
+```yaml
```
Comment (tengqm) [inline content/en/docs/tasks/manage-kubernetes-objects/kustomization.md:262]: Either you use `yaml` and `none` separately for the following two files, or you can wrap the content using `shell`. This content currently is not valid YAML.

Comment (xordspar0) [reply content/en/docs/tasks/manage-kubernetes-objects/kustomization.md]: You're right, good catch. Fixed.

**Comment by xordspar0** [comment]
Is there a different approver who could look at this? steveperry-53 hasn't been active for a while.

**Comment by sftim** [comment]
Sorry about the wait @xordspar0 - we've been [busy](https://blog.k8s.io/2021/04/08/kubernetes-1-21-release-announcement/) just recently.

**Comment by xordspar0** [comment]
No problem at all, and congrats on the release!

**Pull request merged**